ember_debug/view-debug.js

```diff
@@ -334,7 +334,7 @@ export default EmberObject.extend(PortMixin, {
     if (!emberApp) {
       return false;
     }
-    let applicationView = document.querySelector(`${emberApp.rootElement} > .ember-view`);
+    let applicationView = document.querySelector(`${emberApp.rootElement} > [class='ember-view']`);
```

Comment:
Hmm, I think this is still somewhat broken. Specifically when the `application-template-wrapper` optional feature is enabled, there will be no root wrapping div at all.

https://github.com/emberjs/rfcs/pull/280 for references

Reply:
Hmm, wait, is this only supporting legacy cases? Seems like everything 2.10+ (including anyone witht he optional feature) would be hitting the this.glimmerTree branch, right?

Reply:
@rwjblue you are correct, anything glimmer 2 and up should be hitting that `if`. I happen to be working in an app in a version afflicted by this issue, due to liquid-fire. I think for these legacy cases, this should be sufficient. We'll probably want to focus more on the no application wrapper format going forward anyway.

Reply:
We could also follow up with another PR to the 0.0.0-2.7.0 branch to apply this same fix there https://github.com/emberjs/ember-inspector/blob/ember-0.0.0-2.7.0/ember_debug/view-debug.js#L255

Reply:
I think the only case that may throw things off here, is if you were to be manually applying extra classes to the application wrapper div, but I think I am okay with that? I'm also okay with closing this PR and closing #419 as wontfix 😄

Reply:
> manually applying extra classes to the application wrapper div, but I think I am okay with that?

Ya, that seems very suspect and not something I'm terribly worried about supporting by default.

Reply:
@rwjblue are we good to go here then? I will open a PR for the 0.0.0-2.7.0 branch later as well.

Reply:
yep, seems good to me! sorry for my confusion before...

Reply:
@rwjblue no worries! Mind approving real quick then?